### DISCUSSION TITLE
Copy/Paste without formating

### DISCUSSION THREAD

**Discussion by wneessen** [Eugeny/tabby#6056] [Q&A] [answered]
Hi,

is there a way to copy from Tabby without formating. Sometimes, especially when trying to copy special characters and then pasting into MacOS Mail.app it wants to apply some magic, that completely breaks the paste action. Also having colored text when I just want some plain text pasted is very annyoing. It for example copies the background color of the marked text and that is then pasted as well. Would be great to know if there is some setting or something I can do to avoid formated copy/pasting.

Example:

Tabby => Mail.app

<img width="898" alt="Screenshot 2022-03-30 at 14 40 19" src="https://user-images.githubusercontent.com/542696/160836533-063a9348-157e-409c-8d87-fea5c0c347d3.png">

<img width="680" alt="Screenshot 2022-03-30 at 14 40 44" src="https://user-images.githubusercontent.com/542696/160836558-874c5097-7aeb-4d20-83ac-2adb0aa30080.png">

**Comment by Eugeny** [comment, marked as answer]
Here's the issue for it: #5050 

As a workaround, you can also "Paste and match style" almost anywhere in macOS (Cmd-Shift-V or Cmd-Shift-Alt-V)

**Reply by wneessen** [reply]
Weird. I explicitly searched the open issues before opening the topic...  Must simply have overlooked it. Thanks for the swift reply and looking forward for the feature update.

**Reply by lawweiliang** [reply]
For other,
Setting -> Terminal -> Clicpboard Section -> Click the Toggle button for 'Copy with formatting'

![image](https://user-images.githubusercontent.com/34936929/228127708-03ec36bb-5952-495d-bb0a-25b7b2ea45f2.png)

**Comment by lawweiliang** [comment]
@wneessen 
Please close the discussion if the question was answered. 
It would be helpful to make the discussion section clean. 

Click `mark as answered` and `close the discussion`. 😄 

![image](https://user-images.githubusercontent.com/34936929/228098333-222ac3d4-4c39-4c26-a2a8-52906647e00e.png)
